test/regression_tests.sh

```diff
@@ -29,7 +29,7 @@ install_torchserve_from_source() {
   echo "Installing torchserve torch-model-archiver from source"
   ./scripts/install_from_src_ubuntu
   echo "Torchserve Succesfully installed"
-  
+  pip install transformers 
```

Comment:
Can you please chain this lib installation to a previous pip install command.

https://github.com/HamidShojanazeri/serve/blob/b2e54c44ed369eb299a08079bbba406715458873/test/regression_tests.sh#L18

Reply:
This looks good to me otherwise.

Reply:
Thanks a lot @dhanainme, it's modified as suggested.